crouton.php

```diff
@@ -874,6 +875,16 @@ function admin_options_page() {
 					echo "<div class='updated'><p>Note: consider Deleting Cache (unless you know what you're doing)</p></div>";
 				}
 			}
+
+			if ( !isset($this->options['extra_meetings_enabled']) || strlen(trim($this->options['extra_meetings_enabled'])) == 0) {
+				$this->options['extra_meetings_enabled'] = 1;
```

Comment:
this should be `0` not `1`

Reply:
that makes sense, should I do another PR or just fix in master

Reply:
new PR